controllers/v1/events/event.controller.js

```diff
@@ -45,19 +47,10 @@ const post = async (request, response) => {
   ) {
     return response.status(400).json(baseController.getErrorResponse('Parameters are missing'))
   }
-
-  eventData = {
-    name: request.body.name,
-    date: request.body.date,
-    headquarter: request.body.headquarter,
-    responsable: request.body.responsable,
-    images: [],
-  }
-
-  eventData['address'] = request.body.address ? request.body.address : ''
-  eventData['placeName'] = request.body.placeName ? request.body.placeName : ''
-
-  eventData['phoneNumber'] = request.body.phoneNumber ? request.body.phoneNumber : ''
+  const newEvent = request.body
+  ACTIVE_DB === DEFAULT_DB
+  ? (eventData = mongoEventData(newEvent))
+  : (eventData = firebaseEventData(newEvent))
```

Comment:
Creo q esto se lee mejor
```suggestion
  eventData = firebaseEventData(newEvent)
  if(ACTIVE_DB === DEFAULT_DB) {
    eventData = mongoEventData(newEvent)
  }
  
```